runtime/vm/jniinv.c

```diff
@@ -1257,14 +1257,20 @@ jint JNICALL ResetJavaVM(JavaVM *vm)
 /* run the shutdown method in java.lang.Shutdown */
 void 
 sidecarExit(J9VMThread* shutdownThread) {
-	UDATA args[] = {130};	/* The exit code */
-	J9NameAndSignature nas;
+	/* static void java.lang.Shutdown.exit(int status)
+	 *
+	 * The exit code is 130. Store ints in the low-memory half
```

Comment:
I still don't think the comment matches the code. The original code was storing into the low-memory half of a 64-bit value. The new code is always storing into the first 32-bit, which would be the upper-memory half of a 64-bit value on big endian.

Reply:
Actually its my interpretation of "low-memory" that is the problem. Technically the comment is correct.